src/realm/util/file.cpp

```diff
@@ -223,6 +192,10 @@ void make_dir(const std::string& path)
 
 void make_dir_recursive(std::string path)
 {
+#if REALM_HAVE_STD_FILESYSTEM
+    std::error_code ignored;
```

Comment:
Do we really want to ignore errors here? That feels weird bue matches the other impl so 🤷

Reply:
Actually on a second read the other implementation will throw errors (the POSIX leg `try_make_dir` of does throw). I will change the `std::filesystem` leg to match.